### PR TITLE
Release 9.2.2

### PR DESCRIPTION
**Pull request by MicK7**
update with new available release

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by MicK7** [comment]
@conda-forge-admin, please rerender

**Comment by MicK7** [comment]
@basnijholt ready for merge review

**Comment by basnijholt** [comment]
@conda-forge-admin, please restart ci

File: recipe/bld.bat [hunk 1]
```diff
@@ -32,6 +32,7 @@ cmake .. -G "Ninja" ^
     -DVTK_MODULE_USE_EXTERNAL_VTK_fmt:BOOL=OFF ^
     -DVTK_MODULE_USE_EXTERNAL_VTK_cgns:BOOL=OFF ^
     -DVTK_MODULE_USE_EXTERNAL_VTK_ioss:BOOL=OFF ^
+    -DVTK_MODULE_USE_EXTERNAL_VTK_verdict:BOOL=OFF ^
```
Comment (basnijholt) [inline recipe/bld.bat:35]: Why did this change?

Comment (MicK7) [reply recipe/bld.bat:35]: This is needed for vtk to configure and compile correctly.
From my understanding, the global VTK_MODULE_USE_EXTERNAL was switch from OFF to ON at some point and the verdict library was not included at this time. Now, the library is required and not packaged anywhere on conda, thus to make things work the easiest way was to tell vtk to use its own version of the library.
Notice that there is a will to remove vendored copy #55 and verdict is tagged as part of the job.

**Pull request merged**